.gitignore

```diff
@@ -4,3 +4,5 @@
 dist/
 coverage/
 node_modules/
+yarn.lock
+cypress/integration/examples
```

Comment:
@obulat  No whenever we run `yarn run test:cypress` it will automatically regenerate. And possibilities are high that a new contributior can commit it .

Reply:
I am currently testing this. One note I want to add: The chooser uses npm as a package manager. It is advised not to mix yarn and npm to avoid inconsistencies.

Reply:
@Cronus1007 , I just used yarn to install all the packages and run cypress tests, and the 'cypress/integration/examples' was not regenerated. So, it is safe to remove it from gitignore

Reply:
Okk